src/core/util.ts

```diff
@@ -24,7 +24,28 @@ export const HexStr = {
   isHex(value: any, length?: number): boolean {
     return bytes.isHexString(value, length);
   },
+  isSameHex( a:string, b:string ): boolean {
+    if ( this.isHex(a) && this.isHex(b) ) {
+      let A = String(a).toLocaleUpperCase();
+      let B = String(b).toLocaleUpperCase();
+      return A.localeCompare( B.toString() ) == 0; 
+    } else {
+      return false; 
+    }
+  },
   stripZeros(value: any): string {
     return bytes.hexlify(bytes.stripZeros(value));
+  },
+  zeroPad(value:string, length:number): string {
```

Comment:
There's ethers.utils.hexZeroPad https://docs.ethers.org/v5/api/utils/bytes/#utils-hexZeroPad
If possible, use ethers library for consistency.

Reply:
Good. I will update in web3klaytn.